wen-restart/src/wen_restart.rs

```diff
@@ -154,34 +154,37 @@ pub(crate) fn aggregate_restart_last_voted_fork_slots(
                     .insert(from, record);
             }
         }
-        let result = last_voted_fork_slots_aggregate.get_aggregate_result();
+        // Because all operations on the aggregate are called from this single thread, we can
+        // fetch all results separately without worrying about them being out of sync. We can
+        // also use returned iterator without the vector changing underneath us.
+        let active_percent = last_voted_fork_slots_aggregate.active_percent();
+        let slots_to_repair = last_voted_fork_slots_aggregate.slots_to_repair();
```

Comment:
nit: I think we can just call `last_voted_fork_slots_aggregate.slots_to_repair()` in the filter below instead of saving it here

Reply:
Done.